Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
```

Comment:
@sthistle Other than hardcoding the RGB values, is there a more elegant way to associate colorPrimary with the salesforce_brand_secondary, for example?